internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -40,11 +40,13 @@ const (
 	ErrorAdvancedClusterListStatus = "error awaiting MongoDB ClusterAdvanced List IDLE: %s"
 	ErrorOperationNotPermitted     = "error operation not permitted"
 	ignoreLabel                    = "Infrastructure Tool"
+	DeprecationOldSchemaAction     = "Please refer to our examples, documentation, and 1.18.0 migration guide for more details"
```

Comment:
should we have a link to somewhere here?

Reply:
perhaps to the migration guide?

Reply:
added link to migration guide###